TreeTracker/UI/Views/Profile Screen/ProfileViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ProfileViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 3/23/21.
+//  Copyright © 2021 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+    var trees: [Tree] = []
```

Comment:
I don't think we need to hold onto any tree objects in this view

Reply:
I removed the variable trees from ProfileViewController